lua/lspconfig/server_configurations/stimulus_ls.lua

```diff
@@ -0,0 +1,26 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    cmd = { 'stimulus-language-server', '--stdio' },
+    filetypes = { 'html', 'ruby', 'eruby', 'haml', 'slim', 'php' },
```

Comment:
> This Language Server works for HTML and ERB files

I just saw this on this server README. so why there have many filetypes. related ?

Reply:
> I just saw this on this server README. so why there have many filetypes. related ?

I think we would be fine if we just added these `filetypes` for now:

```lua
filetypes = { 'html', 'ruby', 'eruby', 'blade', 'php' },
```